src/App.js

```diff
@@ -7,8 +7,30 @@ import ToDoForm from "./components/ToDoForm";
 import "./App.css";
 
 
+
 function App() {
-    const [toDoList, setToDoList] = useState(data);
+    const [toDoList, setToDoList] = useState([]);
+
+    useEffect(() => {
+        getStateFromLocalStorage()
+    }, [])
+
+    const getStateFromLocalStorage = () =>{
+        let statedata = localStorage.getItem('state');
```

Comment:
Please use const for variables that have value assigned only once when declaring the variable.